notebooks/02_model/lightfm_deep_dive.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<p>can you please make sure that these formulas are rendered correctly on Github:</p><pre class="ql-syntax" spellcheck="false">\begin{equation}&nbsp;q_u = \sum_{j \in f_u} e_{j}^{U}&nbsp;\end{equation}

\begin{equation}&nbsp;p_i = \sum_{j \in f_i} e_{j}^{I}&nbsp;\end{equation}

Similarly the biases for user&nbsp;

\begin{equation}&nbsp;b_u = \sum_{j \in f_u} b_{j}^{U}&nbsp;\end{equation}

\begin{equation}&nbsp;b_i = \sum_{j \in f_i} b_{j}^{I}&nbsp;\end{equation}
</pre><p>I don't see them right</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
fixed